client/src/apis/chessClient.sockets.js

```diff
@@ -0,0 +1,31 @@
+import IO from 'socket.io-client';
+
+let socket;
+
+const createConnection = () => {
+  socket = IO.connect('http://localhost:3001');
```

Comment:
please set this up to pick up path and port from config file. I think we already have infrastructure set up to use configs.
Also make sure it is production ready, because this will only work with "localhost"

Reply:
Done